src/Paginator.php

```diff
@@ -168,13 +196,14 @@ public static function setGlobalConfig($config)
 
         static::$config = $config;
 
-        if (isset($config['scrolling_style_plugins'])
+        if (
+            isset($config['scrolling_style_plugins'])
```

Comment:
Looks strange.

Reply:
That's part of laminas-coding-standard v2. If it's changed here, it's because of that.

Reply:
You are right:

> Expressions in parentheses MAY be split across multiple lines, where each subsequent line is indented at least once. When doing so, the first condition MUST be on the next line.